doc/release/release_0.20.rst

```diff
@@ -0,0 +1,539 @@
+  Previously, int32 was used.
+  (`#6342 <https://github.com/scikit-image/scikit-image/pull/6342>`_).
+- Use the minimal required unsigned integer size in ``skimage.filters.rank_order`` which allows to operate the function with higher precision or on larger arrays.
+  Previously, the returned ``labels`` and and ``original_values`` were always of type uint32.
+  (`#6342 <https://github.com/scikit-image/scikit-image/pull/6342>`_).
+
+Changes and new deprecations
+----------------------------
+- Rewrite ``skimage.filters.meijering``, ``skimage.filters.sato``,
+  ``skimage.filters.frangi``, and ``skimage.filters.hessian`` to match the published algorithms more closely.
+  This change is backward incompatible and will lead to different output values compared to the previous implementation.
+  The Hessian matrix calculation is now done more accurately.
+  The filters will now correctly be set to zero whenever one of the hessian eigenvalues has a sign which is incompatible with a ridge of the desired polarity.
+  The gamma constant of the Frangi filter is now set adaptively based on the maximum Hessian norm
+  (`#6446 <https://github.com/scikit-image/scikit-image/pull/6446>`_).
+- Return ``False`` in ``skimage.measure.LineModelND.estimate`` instead of raising an error if the model is under-determined
+  (`#6453 <https://github.com/scikit-image/scikit-image/pull/6453>`_).
+- Return ``False`` in ``skimage.measure.CircleModel.estimate`` instead of warning if the model is under-determined
+  (`#6453 <https://github.com/scikit-image/scikit-image/pull/6453>`_).
+- Rename ``skimage.filter.inverse`` to ``skimage.filter.inverse_filter``.
+  ``skimage.filter.inverse`` is deprecated and will be removed in the next release
+  (`#6418 <https://github.com/scikit-image/scikit-image/pull/6418>`_).
+- Update minimal supported dependencies to ``numpy>=1.20``
+  (`#6565 <https://github.com/scikit-image/scikit-image/pull/6565>`_).
+- Update minimal supported dependencies to ``scipy>=1.8``
+  (`#6564 <https://github.com/scikit-image/scikit-image/pull/6564>`_).
+- Update minimal supported dependencies to ``networkx>=2.8``
+  (`#6564 <https://github.com/scikit-image/scikit-image/pull/6564>`_).
+- Update minimal supported dependency to ``pillow>=9.0.1``
+  (`#6402 <https://github.com/scikit-image/scikit-image/pull/6402>`_).
+- Update optional, minimal supported dependency to ``matplotlib>=3.3``
+  (`#6383 <https://github.com/scikit-image/scikit-image/pull/6383>`_).
+- Warn for non-integer image inputs to ``skimage.feature.local_binary_pattern``.
+  Applying the function to floating-point images may give unexpected results when small numerical differences between adjacent pixels are present
+  (`#6272 <https://github.com/scikit-image/scikit-image/pull/6272>`_).
+
+Completed deprecations
+----------------------
+- Remove ``skimage.viewer`` which was scheduled for the postponed version 1.0
```

Comment:
Either "the removal of which" or "which was scheduled for removal on the postponed version 1.0"